python/GafferUITest/NumericWidgetTest.py

```diff
@@ -0,0 +1,113 @@
+############################
```

Comment:
Are we rationing `#` in readiness for a second wave? :)

Reply:
I was totally out. Fortunately I was able to source some from a neighbouring village. Fixed [here](https://github.com/GafferHQ/gaffer/pull/3873/commits/17324a8a53d899189cf26bd5fac4349a410b79b7)